libs/application/templates/new-primary-school/src/lib/messages.ts

```diff
@@ -363,6 +363,54 @@ export const newPrimarySchoolMessages: MessageDir = {
       defaultMessage: 'Notkun myndefnis',
       description: 'Use of footage',
     },
+
+    photographySection: {
+      id: 'dess.nps.application:childrenNParents.photography.section',
+      defaultMessage: 'Notkun myndefnis',
+      description: 'Usage of footage',
+    },
+    photography: {
+      id: 'dess.nps.application:childrenNParents.photography',
+      defaultMessage:
+        'Samþykki vegna myndatöku, myndbandsupptöku og birtingu myndefnis grunnskóla',
+      description:
+        'Consent for photography, video recording and publication of elementary school footage',
+    },
+    photographyDescription: {
+      id: 'dess.nps.application:childrenNParents.photographyDescription',
+      defaultMessage:
+        'Þegar kemur að myndatöku og myndbirtingu skal virða sjálfsákvörðunarrétt barna og ungmenna og taka tillit til þeirra skoðana þeirra og viðhorfa í samræmi við aldur og þroska.',
```

Comment:
Can you update this text?

```suggestion
        'Þegar kemur að myndatöku og myndbirtingu skal virða sjálfsákvörðunarrétt barna og ungmenna og taka tillit til skoðana og viðhorfa þeirra í samræmi við aldur og þroska.',
```